hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/bucket/BucketStreamWriteFunction.java

```diff
@@ -157,8 +157,8 @@ private void bootstrapIndexIfNeed(String partition) {
 
     // Load existing fileID belongs to this task
```

Comment:
Does this comment need to change to `Load the latest fileID belongs to this task`?

Reply:
Not really, added comments on when we cannot load the latest fileId (latest fileID = committed fileId)